src/web/app/components/feedback-path-panel/feedback-path-panel.component.spec.ts

```diff
@@ -26,10 +27,59 @@ describe('FeedbackPathPanelComponent', () => {
   beforeEach(() => {
     fixture = TestBed.createComponent(FeedbackPathPanelComponent);
     component = fixture.componentInstance;
+
+    component.customNumberOfEntitiesToGiveFeedbackTo.subscribe((value) => {
+      component.model.customNumberOfEntitiesToGiveFeedbackTo = value;
+    });
+    component.numberOfEntitiesToGiveFeedbackToSetting.subscribe((value) => {
+      component.model.numberOfEntitiesToGiveFeedbackToSetting = value;
+    });
+    component.customFeedbackPath.subscribe((value) => {
+      component.model.isUsingOtherFeedbackPath = value;
+    });
+
     fixture.detectChanges();
   });
 
   it('should create', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should toggle sub menu status correctly', () => {
+    const menu = FeedbackParticipantType.STUDENTS;
+    component.toggleSubMenu(menu);
+    const firstToggle = component.isSubMenuOpen(menu);
+    component.toggleSubMenu(menu);
+    const secondToggle = component.isSubMenuOpen(menu);
+    expect(firstToggle).toBe(!secondToggle);
+  });
+
+  it('should reset menu', () => {
+    const menu = FeedbackParticipantType.STUDENTS;
+    component.toggleSubMenu(menu);
+    component.resetMenu();
+    const isMenuOpen = component.isSubMenuOpen(menu);
+    expect(isMenuOpen).toBe(false);
+  });
+
+  it('should trigger custom number of entities', () => {
+    const customNumber = 5;
+    component.triggerCustomNumberOfEntities(customNumber);
+    const result = component.model.customNumberOfEntitiesToGiveFeedbackTo;
+    expect(result).toEqual(customNumber);
```

Comment:
```suggestion
    const emitSpy = jest.spyOn(component.customNumberOfEntitiesToGiveFeedbackTo, 'emit');
    expect(emitSpy).toHaveBeenCalledWith(customNumber);
```
let's test the emit event this way, same goes for the rest of the tests